Allure.NUnit.AsyncTests/AsyncOneTimeSetup.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Allure.Net.Commons;
+using NUnit.Framework;
+
+namespace Allure.NUnit.AsyncTests
+{
+    [SetUpFixture]
+    public class AsyncOneTimeSetup
+    {
+        [OneTimeSetUp]
+        public void AsyncSetup()
+        {
+            AllureLifecycle.Instance.CleanupResultDirectory();
+
+            //!_! This is essential for async tests, because they are working in different threads and default ThreadStatic<...> behaviour fails.
+            AllureLifecycle.CurrentTestIdGetter = () => TestContext.CurrentContext.Test.FullName;
```

Comment:
Better solution is to provide some base class to extend for this case.
For example, end-user have to write something like:
```csharp
public class MyAwesomeTestClass : AsyncTestBase
```
with all under-the-hood tasks done inside the AsyncTestBase

Reply:
Inheritance is a solution, but in some cases we can see that user's test classes are already inherited from some base class...
Maybe it is a good idea to move this under-the-hood initialization to the static constructor of an `[AllureNUnit]` attribute?